toolchain/binutils.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+backend = SoftwareManager()
```

Comment:
Global variables should use UPPER_CASE notation. Still I think it'd be better to move this inside the `setUp` as it's only used there and having it in the main level makes it prone to import errors.